src/service/price_oracle.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package helium.price_oracle;
+
+import "price_oracle.proto";
+
+// request price oracle
+message price_oracle_req_v1 {}
+
+service price_oracle {
```

Comment:
We can obviously add it later but do we foresee the need to have a price streaming feed to be reactive to price changes; such as swings outside of some defined threshold?

Reply:
That's what the source of the price feed (pyth)  already does. This service just makes it 1. accessible to oracles without needing to include Solana/Pyth sdks  and 2. more resilient to Solana outages and 3. delivers historical data by putting price reports in a bucket for later analysis/use/graphing